### DISCUSSION TITLE
Change the color of the name conditionally

### DISCUSSION THREAD

**Discussion by cokeiill** [Clooos/Bubble-Card#1093] [Questions about config, custom styles and templates] [answered]
Thank you, author，I need to modify the color of the name based on the status of the entity, which cannot be implemented in this code. It may not be supported yet. Do you have any plans to add it? Thank you again for your contribution. Thank you
![image](https://github.com/user-attachments/assets/2f9384ac-68e6-4abc-85d8-679867d50cd6)

**Comment by MrBearPresident** [comment]
For me it is not completly clear what you are trying to do. Could you try to explain with a screenshot?

**Comment by cokeiill** [comment]
What I mean is that the name of the card display can change color according to the state of the device, similar to changing the background color of the card according to the state of the device, because I set the background of the card to be close to white, but the font is still white. I hope the font can change to black when I open the device, or I can customize the color of the card

**Comment by MrBearPresident** [comment, marked as answer]
Try this in your custom-styles: 

```
.bubble-button-background{
  opacity: 1 !important;
  background-color: ${state === 'on'? '#f3e4c6' : '#191818' } !important;
}

.bubble-name-container{
  color: ${state === 'on'? 'black' : 'white' } !important;
}
```

---

<sup> If this solves you issue, please mark this as the solution and close the discussion. </sup>

**Reply by cokeiill** [reply]
Thank you for your reply. It has been perfectly achieved
